.github/prereleases/3-comment.mjs

```diff
@@ -0,0 +1,90 @@
+import assert from "node:assert";
+import fs from "node:fs";
+import path from "node:path";
+import { DefaultArtifactClient } from "@actions/artifact";
+import {
+	getPackagesForPrerelease,
+	getPrereleaseArtifactUrl,
+	getPrereleasePRArtifactUrl,
+	projectRoot,
+} from "./0-packages.mjs";
+
+const artifact = new DefaultArtifactClient();
+
+/**
+ * @param {~Package} pkg
+ * @returns {string}
+ */
+function buildWranglerArtifactReport(pkg) {
+	const name = pkg.json.name;
+	assert.strictEqual(name, "wrangler");
+	const url = getPrereleaseArtifactUrl(name);
+	const prUrl = getPrereleasePRArtifactUrl(name);
+	return `A wrangler prerelease is available for testing. You can install this latest build in your project with:
```

Comment:
I am not a big fan of the markdown being all included here like so (and all the backticks escapes)

I am fine with it, but hopefully we can come up with something more convenient at some point 🙂

Reply:
Also I've disliked the prerelease comment for a while and I did try to update it 😅

It's not something really related to this PR, but yeah I'd love it if we could improve the comment/report as a whole, I just figured I'd mention it here 😄

Reply:
Yes, I was thinking of something like handlebars as well 🙂

But it was just a thought, I actually don't think we should overthink this in this PR and increase it's scope, I'm 100% fine with the current implementation, I just wanted to mention that it'd be nice maybe later at some point to improve things 🙂